docs/docs/tracing/minimal-sdk.mdx

```diff
@@ -0,0 +1,215 @@
+    />
+    <TitleCard
+        title="🔒 Fewer Security Risks"
+        description="Each dependency potentially introduces security vulnerabilities. By reducing the number of dependencies, MLflow Tracing minimizes the attack surface and reduces the risk of security breaches."
+    />
+</CardGroup>
+
+## Installation
+
+Install the lightweight SDK using pip:
+
+```bash
+pip install mlflow-tracing
+```
+
+:::warning
+
+Do not install the full `mlflow` package together with the lightweight SDK, as this may cause version conflicts.
+
+:::
+
+## Quickstart
+
+Here's a simple example using the lightweight SDK with OpenAI for logging traces to an experiment
+on a remote MLflow server. If you haven't set up a remote MLflow server yet, please refer to the [Choose Your Backend](#choose-your-backend) section below.
+
+
+```python
+import mlflow
+import openai
+
+# Set the tracking URI (e.g., Databricks, SageMaker, or self-hosted server)
+mlflow.set_tracking_uri("<your-tracking-uri>")
+mlflow.set_experiment("<your-experiment-name>")
+
+# Enable auto-tracing for OpenAI
+mlflow.openai.autolog()
+
+# Use OpenAI as usual
+client = openai.OpenAI()
+response = client.chat.completions.create(
+    model="gpt-4", messages=[{"role": "user", "content": "What is MLflow?"}]
+)
+```
+
+
+## Choose Your Backend
+
+The lightweight SDK works with various observability platforms. Choose your favorite one and follow the instructions to set up your tracing backend.
+
+<Tabs>
+  <TabItem value="databricks-monitoring" label="Databricks">
+    <div class="flex-column">
+      <div class="flex-row">
+        <div class="flex-item">
+
+          [Databricks Lakehouse Monitoring for GenAI](https://docs.databricks.com/aws/en/generative-ai/agent-evaluation/monitoring) is a go-to solution for monitoring your
+          GenAI application with MLflow Tracing. It provides an instant access to a fully functional monitoring dashboard for operational excellence and quality analysis.
+
+          Lakehouse Monitoring for GenAI can be used regardless of whether your application is hosted on Databricks or not.
+
+          [Sign up for free](https://docs.databricks.com/aws/en/getting-started/free-trial) and [get started in a minute](https://docs.databricks.com/aws/en/generative-ai/agent-evaluation/monitoring) to run your GenAI application with complete observability.
+        </div>
+
+        <div class="flex-item padding-md">
+          ![Monitoring](https://assets.docs.databricks.com/_static/images/generative-ai/monitoring/monitoring-hero.gif)
+        </div>
+      </div>
+    </div>
+
+  </TabItem>
+  <TabItem value="self-hosting" label="Self-hosting">
+    <div class="flex-column">
+      <div class="flex-row">
+        <div class="flex-item">
+
+          MLflow is a **fully open-source project**, allowing you to self-host your own MLflow server and keep your data private. This is a great option if you want to have full control over your data and infrastructure.
```

Comment:
```suggestion
          MLflow is a **fully open-source project**, allowing you to self-host your own MLflow server in your own infrastructure. This is a great option if you want to have full control over your data and are restricted in using cloud-based services.
```
Let's not mention "keeping data private". This implies that storage in cloud based services of MLflow isn't secure, which is not true. Most large-scale self-hosting is due to restrictions about keeping service on on-prem data centers rather than a fear that a provider can't keep data isolated.